WordPress/Classes/ViewRelated/Site Creation/Wizard/SiteCreationWizardLauncher.swift

```diff
@@ -7,7 +7,8 @@ final class SiteCreationWizardLauncher {
     private lazy var segmentsStep: WizardStep = {
         // It seems like we might need to pass the builder to all the steps. I am not too fond of inheritance in general, but this might make a good case for a base Step.
         // Actually, the address site looks exactly like this, so a generics-based approach might actually work better. I'll reconsider this setup after implementing the address step
-        return SiteSegmentsStep(creator: self.creator, service: MockSiteSegmentsService())
+        let segmentsService = SiteCreationSegmentsService(managedObjectContext: ContextManager.sharedInstance().mainContext)
```

Comment:
As we near the end of this project, is it worth removing the comment above, or perhaps extracting it and adding a "tech debt" or similar refactoring issue to address this in the future?